tutorial/1. user_tutorial.ipynb

```diff
@@ -0,0 +1,665 @@
+    "For extended documentation about the Species class, go [here](https://ncar.github.io/musica/api/python.html#musica.mechanism_configuration.Species).</br>\n",
+    "Here is an example function of a basic Arrhenius reaction:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "654b7e30",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def define_species(): # Create each of the species that will be simulated\n",
+    "    X = mc.Species(name=\"X\")\n",
+    "    Y = mc.Species(name=\"Y\")\n",
+    "    Z = mc.Species(name=\"Z\")\n",
+    "    species = {\"X\": X, \"Y\": Y, \"Z\": Z}\n",
+    "    gas = mc.Phase(name=\"gas\", species=list(species.values()))\n",
+    "    return gas, species"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9e6190b7",
+   "metadata": {},
+   "source": [
+    "The <i>define_system()</i> function creates 3 species called X, Y, and Z and adds them to a dictionary called species.</br>\n",
+    "The variable name and the name of the species are not required to be the same, but it is strongly recommended so that your code is more organized.</br>\n",
+    "Next, a gas object is created to represent the phase of the species and then the gas and species objects are returned.</br>\n",
+    "<b>Note:</b> Returning the gas and species containers is mandatory, as subsequent code relies on the data being bundled in specific object types."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1afce428",
+   "metadata": {},
+   "source": [
+    "## 2. Defining Reactions\n",
+    "\n",
+    "A chemical reaction is a process that results into a set of reactants transforming into a set of products.</br>\n",
+    "To define a reaction in MusicBox:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bf790275",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def define_reactions(gas, species): # Create the reactions that the species undergo in the\n",
+    "    arr1 = mc.Arrhenius(name=\"X->Y\", A=4.0e-3, C=50, reactants=[species[\"X\"]], products=[species[\"Y\"]], gas_phase=gas)\n",
+    "    arr2 = mc.Arrhenius(name=\"Y->Z\", A=4.0e-3, C=50, reactants=[species[\"Y\"]], products=[species[\"Z\"]], gas_phase=gas)\n",
+    "    rxns = {\"X->Y\": arr1, \"Y->Z\": arr2}\n",
+    "    return rxns"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "00cb442b",
+   "metadata": {},
+   "source": [
+    "The <i>define_reaction()</i> function takes in the gas and species variables returned from the <i>define_system()</i> function.</br>\n",
+    "Using the species and gas variables, the function creates two reactions: arr1 and arr2.</br>\n",
+    "The arr1 variable represents the conversion of the X reacant into the Y product with some pre-defined constants as well.</br>\n",
+    "The arr2 variable is just like arr1, but instead it represents the conversion of the Y reactant into the Z product.</br>\n",
```

Comment:
```suggestion
    "The arr1 variable represents the conversion of X (reactant) into Y (product) and defines Arrhenius rate constant parameters A and C.</br>\n",
    "The arr2 variable is just like arr1, but instead it represents the conversion of Y (reactant) into Z (product).</br>\n",
```
Maybe add a link to the documentation for Arrhenius reactions that shows the equation including the A and C parameters?